composeApp/src/commonMain/kotlin/com/majotyler/hiittimer/presentation/TimerViewModel.kt

```diff
@@ -42,25 +42,23 @@ class TimerViewModel : ViewModel() {
         }
     }
 
-    suspend fun onClickedAdd() {
+    private suspend fun onClickedAdd() {
         _loading.value = true
         delay(4000)
         _loading.value = false
-
-
     }
 
-    fun onClickedDelete(index: Int) {
+    private fun onClickedDelete(index: Int) {
         _exercises.value = _exercises.value.toMutableList().apply {
             removeAt(index)
         }
     }
 
-    fun addReps() {
+    private fun addReps() {
         _reps.value++
     }
 
-    fun removeReps() {
+    private fun removeReps() {
```

Comment:
These should all be `private`, so that `TimerScreen` has to interact with the ViewModel through `onEvent`, which is `public`.